sideboard/config.py

```diff
@@ -12,6 +12,7 @@
 class ConfigurationError(RuntimeError):
```

Comment:
Most of the changes in this file are just minor refactors and cleaning up some backwards-compatibility that no one (including MAGFest) ever actually used.